raisim_gym/env/env/hummingbird/Environment.hpp

```diff
@@ -199,6 +248,33 @@ class ENVIRONMENT : public RaisimGymEnv {
     Eigen::Vector3d force_B;
     force_B << 0.0, 0.0, genForce(3);
 
+    double kp_rot = -0.2, kd_rot = -0.06;
+    Eigen::Vector3d fbtorque_B 
+
+    if (angle > 1e-6){
+        fbtorque_B = kp_rot * rot_angle * (rot_.transpose() * Orientation) 
+            / sin(rot_angle) + kd_rot * angVel_B_;
+    }
+    else
+    {
+      fbtorque_B = kd_rot * angVel_B_;
+      fbtorque_B(2) = fbtorque_B(2) * 0.15; 
+    }
+
+    torque_B += fbtorque_B;  /// sum of torque inputs
+
+    force_B(2) += mass_ * 9.81; /// Sum of thrust inputs
```

Comment:
why do you add gravity? this should be done by raisim

Reply:
oh, the original implementation has this as action scaling. Since we already have it in a different place, you don't need it